File: server/src/main/java/com/example/pametneucionice/server/model/RoomResponse.java
package com.example.pametneucionice.server.model;

import java.util.List;

public record RoomResponse(
        String roomId,
        String roomName,
        boolean occupied,
        String currentClassName,
        String occupiedUntil,
        List<ScheduleEntryResponse> schedule,
        double temperatureCelsius,
        String temperatureStatus,
        double noiseDecibels,
        String noiseStatus,
        int carbonDioxidePpm,
        String airQualityStatus,
        String recommendation
) {
}

File: android/app/src/main/java/com/example/pametneucionice/model/RoomResponse.kt
package com.example.pametneucionice.model

import kotlinx.serialization.Serializable

@Serializable
data class RoomResponse(
    val roomId: String,
    val roomName: String,
    val occupied: Boolean,
    val currentClassName: String?,
    val occupiedUntil: String?,
    val schedule: List<ScheduleEntryResponse>,
    val temperatureCelsius: Double,
    val temperatureStatus: SensorStatus,
    val noiseDecibels: Double,
    val noiseStatus: SensorStatus,
    val carbonDioxidePpm: Double,
    val airQualityStatus: SensorStatus,
    val recommendation: String
)
